mitodl/mit-open
# Load fixtures by default in dev environment

### What are the relevant tickets?

Closes #482 


### Description (What does it do?)
the fixtures that are loaded by running ```manage.py loaddata platforms departments offered_by``` sets up some default data that is needed by other commands such as ```manage.py backpopulate_ocw_data``` and ```manage.py backpopulate_prolearn_data```. This PR adds a change that runs this by default on development and makes fixture loading an optional step.

### How can this be tested?
1. checkout the main branch.
2. make sure your environment is setup to be able to pull course data from the prolearn catalog (set PROLEARN_CATALOG_API_URL=https://prolearn.mit.edu/graphql in .env)
3. bring up your environment via:
```
docker-compose down
docker-compose up
```
4. run ```docker compose run --rm web python manage.py backpopulate_prolearn_data``` and note that it will return with ```Populated 0 resources``` due to default objects not existing in the database
5. checkout this branch (shanbady/482-load-fixture-files-by-default) and repeat steps 2-4
6. note that the backpopulate_prolearn_data command populates properly


### Checklist:
- [ ] Verified this PR via the steps listed in testing section above.
